### PR TITLE
pw-metadata: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/pw-metadata.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+> Vea también: `pipewire`, `pw-mon`, `pw-cli`.
+> Más información: <https://docs.pipewire.org/page_man_pw-metadata_1.html>.
+
+- Muestra metadatos en formato `predeterminado`:
+
+`pw-metadata`
+
+- Muestra metadatos con ID 0 en `configuración`:
+
+`pw-metadata {{-n|--name}} {{configuración}} {{0}}`
```
Comment (spageektti) [inline pages.es/linux/pw-metadata.md:13]:
```suggestion
`pw-metadata {{-n|--name}} {{settings}} {{0}}`
```
I don't think this should be translated as `settings` really exist on systems with pipewire so it's a ready-to-use example.

File: pages.es/linux/pw-metadata.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+`pw-metadata {{-m|--monitor}}`
+
+- Elimina todos los metadatos:
+
+`pw-metadata {{-d|--delete}}`
+
+- Ajusta `log.level` a 1 en `configuración`:
+
+`pw-metadata --name {{configuración}} {{0}} {{log.level}} {{1}}`
```
Comment (spageektti) [inline pages.es/linux/pw-metadata.md:29]:
```suggestion
`pw-metadata --name {{settings}} {{0}} {{log.level}} {{1}}`
```
same as above

File: pages.es/linux/pw-metadata.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Muestra la ayuda:
```
Comment (tricantivu) [inline pages.es/linux/pw-metadata.md:31]:
```suggestion
- Muestra ayuda:
```

File: pages.es/linux/pw-metadata.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Muestra metadatos con ID 0 en `configuración`:
+
+`pw-metadata {{-n|--name}} {{configuración}} {{0}}`
+
+- Lista todos los objetos de metadatos disponibles:
+
+`pw-metadata {{-l|--list}}`
+
+- Sigue ejecutando y registrando los cambios en los metadatos:
```
Comment (tricantivu) [inline pages.es/linux/pw-metadata.md:19]:
```suggestion
- Continua ejecutando y registrando los cambios en los metadatos:
```

File: pages.es/linux/pw-metadata.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+> Supervisa, establece y elimina metadatos en objetos PipeWire.
+> Vea también: `pipewire`, `pw-mon`, `pw-cli`.
+> Más información: <https://docs.pipewire.org/page_man_pw-metadata_1.html>.
+
+- Muestra metadatos en formato `predeterminado`:
+
+`pw-metadata`
+
+- Muestra metadatos con ID 0 en `configuración`:
```
Comment (tricantivu) [inline pages.es/linux/pw-metadata.md:11]:
```suggestion
- Muestra metadatos con el identificador 0 en `configuración`:
```

File: pages.es/linux/pw-metadata.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# pw-metadata
+
+> Supervisa, establece y elimina metadatos en objetos PipeWire.
+> Vea también: `pipewire`, `pw-mon`, `pw-cli`.
+> Más información: <https://docs.pipewire.org/page_man_pw-metadata_1.html>.
+
+- Muestra metadatos en formato `predeterminado`:
```
Comment (tricantivu) [inline pages.es/linux/pw-metadata.md:7]:
```suggestion
- Muestra metadatos en el formato por defecto:
```

File: pages.es/linux/pw-metadata.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+> Supervisa, establece y elimina metadatos en objetos PipeWire.
+> Vea también: `pipewire`, `pw-mon`, `pw-cli`.
+> Más información: <https://docs.pipewire.org/page_man_pw-metadata_1.html>.
+
+- Muestra metadatos en el formato por defecto:
+
+`pw-metadata`
+
+- Muestra metadatos con el identificador 0 en `configuración`:
```
Comment (spageektti) [inline pages.es/linux/pw-metadata.md:11]:
```suggestion
- Muestra metadatos con el identificador 0 en `settings`:
```

File: pages.es/linux/pw-metadata.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+- Continua ejecutando y registrando los cambios en los metadatos:
+
+`pw-metadata {{-m|--monitor}}`
+
+- Elimina todos los metadatos:
+
+`pw-metadata {{-d|--delete}}`
+
+- Ajusta `log.level` a 1 en `configuración`:
```
Comment (spageektti) [inline pages.es/linux/pw-metadata.md:27]:
```suggestion
- Ajusta `log.level` a 1 en `settings`:
```

**Review by spageektti — APPROVED** [review]
Descriptions should be updated too.

**Pull request merged**